freestar-plugin-react-native.podspec

```diff
@@ -15,6 +15,6 @@ Pod::Spec.new do |s|
   s.source       = { :git => "https://gitlab.com/freestarcapital/freestar-plugin-react-native.git", :tag => "v#{s.version}" }
   s.source_files  = "ios/**/*.{h,m}"
 
-  s.dependency 'FreestarAds', '~> 4.0'
+  s.dependency 'FreestarAds', '~> 5.14.0'
```

Comment:
@maddapper I have restored to the latest Release version, once we have the Core with the Thumbnail Release ready will update this as well.
FYI @kkawai

Reply:
@carlosalcala-freestar is this version of the plugin compatible with Core 5.14?

Reply:
@maddapper will update when the 5.15 build Release is ready

Reply:
@maddapper Fixed